pysc2/lib/features.py

```diff
@@ -73,6 +74,13 @@ class Effects(enum.IntEnum):
   # pylint: enable=invalid-name
 
 
+class DisplayType(enum.IntEnum):
```

Comment:
This isn't needed. Use the value from raw.proto.

Also, we use 2 space indents.